src/staging/src/templates/park.js

```diff
@@ -147,9 +148,9 @@ export default function ParkTemplate({ data }) {
   const menuItems = [
     {
       sectionIndex: 0,
-      display: capitalizeFirstLetter(`${park.type} overview`),
+      display: capitalizeFirstLetter(`${parkType} overview`),
       link: "#park-overview-container",
-      visible: true,
+      visible: park.description,
```

Comment:
There might be cases where an editor leaves behind a non-zero-length string or an empty `<p></p>` or `<br>` in the RTE, and it would break this condition check.  
You could create a function that uses a regex to remove tags and then trims the string.

Reply:
```
// function to check if a string contains anything besides html tags and whitespace characters
const isNotWhiteSpace = (str) => str.replace(/(<([^>]+)>)|^\s+|\s+$|\s+/g, "").length > 0;
```

`visible: isNotWhiteSpace(park.description)`

Reply:
Thanks! I just copied yours as is 😂

Reply:
Sorry there was a bug in that version where it was choking on null.  Try the new one I posted.  Note that it's inverted now so you have to check for `!`
`!isNullOrWhiteSpace(park.description)`
`!isNullOrWhiteSpace(park.parkOperation)`
`!isNullOrWhiteSpace(park.safetyInfo)`
`!isNullOrWhiteSpace(park.specialNotes)`

Reply:
Thanks! I also applied this function to other `richtext` values, such as `park.parkContact`, `park.natureAndCulture`, and `park.reconciliationNotes`